src/xdocs/config_javadoc.xml

```diff
@@ -3103,29 +3103,49 @@ public class TestClass {
 
       <subsection name="Examples" id="WriteTag_Examples">
         <p>
-        To configure the check for printing author name:
+          To configure the check with default configuration:
```

Comment:
Just write here: "If no properties are specified, the check does nothing."
There is no need for either the default configuration nor the example.